content/develop/interact/search-and-query/advanced-concepts/vectors.md

```diff
@@ -9,369 +9,503 @@ categories:
+| **HNSW**              | Hierarchical Navigable Small Worlds (HNSW) - an optimized version of [nmslib/hnswlib](https://github.com/nmslib/hnswlib), based on [HNSW Graphs](https://arxiv.org/ftp/arxiv/papers/1603/1603.09320.pdf), for scalable vector search over large datasets or for workloads that require extreme scalability. |
 
-         $d(u, v) = 1 -\frac{u \cdot v}{\lVert u \rVert \lVert v  \rVert}$
+### Search capabilities
 
-## Create a vector field
+Redis enables a number of powerful search techniques over vector fields and metadata:
 
-You can add vector fields to the schema in FT.CREATE using this syntax:
+| Search Type             | Description                          |
+|-------------------------|--------------------------------------|
+| **K-nearest neighbors (KNN)** | Search for the top k most similar vectors.  |
+| **Vector range queries**  | Search for vectors within a specified semantic radius.  |
+| **Metadata filtering**  | Filter results based on metadata.  |
+
+
+## Create a vector index
+
+Redis maintains a secondary index on top of your data that has a defined schema. You can include one or more vector fields in your schema using the [`FT.CREATE`]({{< baseurl >}}/commands/ft.create/) command.
+
+**Syntax**
 
 ```
-FT.CREATE ... SCHEMA ... {field_name} VECTOR {algorithm} {count} [{attribute_name} {attribute_value} ...]
+FT.CREATE <index_name>
+  ON <storage_type>
+  PREFIX 1 <key_prefix>
+  SCHEMA ... <field_name> VECTOR <algorithm> <index_attribute_count>
+    [<index_attribute_name> <index_attribute_value> ...]
 ```
 
-Where:
+Refer to the full [indexing]({{< baseurl >}}/develop/interact/search-and-query/indexing/) documentation for additional fields, option, and noted limitations.
+
+**Parameters**
+
+| Parameter         | Description                                                                                       |
+|-------------------|---------------------------------------------------------------------------------------------------|
+| `index_name`      | Name of the index.  |
+| `storage_type`    | Storage option (`HASH` or `JSON`).  |
+| `prefix` (optional) | Key prefix that signals to the index which keys should be included. Defaults to all keys if omitted. |
+| `field_name`      | Name of the vector field.  |
+| `algorithm`       | Vector index algorithm (`FLAT` or `HNSW`).  |
+| `index_attribute_count` | Number of vector field attributes.  |
+| `index_attribute_name`  | Vector field attribute name.  |
+| `index_attribute_value` | Vector field attribute value.  |
 
-* `{algorithm}` must be specified and be a supported vector similarity index algorithm. The supported algorithms are:
 
-    - FLAT - a simple, straightforward algorithm.
+### FLAT index
+**Choose the FLAT index when** you have small datasets (*<1M vectors*) or when search accuracy is most important *even at the expense of additional latency*.
```

Comment:
I like the tips you add in I think that's an improvement and helps orient the reader